INSTALL.md

```diff
@@ -55,7 +55,7 @@ cmake -DFDEEP_BUILD_UNITTEST=ON ..
 make unittest
 cd ../..
 ```
-
+The Unit Tests require Python3 and Tensorflow should be installed as a Python Module. You could use *pip show tensorflow* in terminal or *import tensorflow* in Python to check your Tensorflow.
```

Comment:
The [requirements-and-installation section](https://github.com/Dobiasd/frugally-deep/tree/881c2ae8be6e84b38e40c1cee901f32e54581920#requirements-and-installation) in the README already states that TensorFlow is required. Does this remark add any new information?

Reply:
Yes I just modified it and removed 'tensorflow'